android/pubsub.md

```diff
@@ -68,6 +68,17 @@ Then, you need to attach the `myIOTPolicy` policy to the user's *Cognito Identit
 aws iot attach-principal-policy --policy-name 'myIOTPolicy' --principal '<YOUR_COGNITO_IDENTITY_ID>'
 ```
 
+You can also programmatically attach the `myIOTPolicy` policy to the user's *Cognito Identity Id* as follows :
+
+```
```

Comment:
Please mark this as a Java code block for proper syntax highlighting

Reply:
Done